internal/cluster/cluster_test.go

```diff
@@ -3985,3 +3993,73 @@ var _ = Describe("UploadEvents", func() {
 		Expect(notUploadedCluster2.Uploaded).To(BeTrue())
 	})
 })
+
+var _ = Describe("ResetClusterFiles", func() {
+	var (
+		ctx               = context.Background()
+		db                *gorm.DB
+		dbName            string
+		clusterID         strfmt.UUID
+		hostID            strfmt.UUID
+		capi              API
+		mockManifestsApi  *manifestsapi.MockManifestsAPI
+		mockObjectHandler *s3wrapper.MockAPI
+		ctrl              *gomock.Controller
+	)
+
+	createCluster := func() common.Cluster {
+		clusterID = strfmt.UUID(uuid.New().String())
+		cluster := common.Cluster{
+			Cluster: models.Cluster{
+				ID:     &clusterID,
+				Status: swag.String(models.ClusterStatusReady),
+			},
+		}
+		hostID = strfmt.UUID(uuid.New().String())
+		host := models.Host{
+			ID: &hostID,
+		}
+		cluster.Hosts = append(cluster.Hosts, &host)
+		Expect(db.Create(&cluster).Error).ToNot(HaveOccurred())
+		return cluster
+	}
+
+	BeforeEach(func() {
+		db, dbName = common.PrepareTestDB()
+		dummy := &leader.DummyElector{}
+		ctrl = gomock.NewController(GinkgoT())
+		mockOperators := operators.NewMockAPI(ctrl)
+		mockManifestsApi = manifestsapi.NewMockManifestsAPI(ctrl)
+		mockObjectHandler = s3wrapper.NewMockAPI(ctrl)
+		capi = NewManager(getDefaultConfig(), common.GetTestLog(), db, commontesting.GetDummyNotificationStream(ctrl), nil, nil, nil, nil, nil, dummy, mockOperators, nil, mockObjectHandler, nil, nil, mockManifestsApi)
+	})
+
+	AfterEach(func() {
+		common.DeleteTestDB(db, dbName)
+		ctrl.Finish()
+	})
+
+	It("Should clear only user generated manifests when performing a reset", func() {
```

Comment:
Isn't this test name incorrect?
It should clear only system manifests when resetting, right?

Reply:
Yes, will change this